decompy/ai-rl/RewardFunction.cpp

```diff
@@ -0,0 +1,205 @@
+#include "llvm/IR/Function.h"
```

Comment:
I wonder if Sphinx can detect comments for C++ as well. Then we can do javadoc like commenting styles for this.

Reply:
We don't have smth set up for that yet  so just ignore it for now